app/controllers/dashboard/protocols_controller.rb

```diff
@@ -218,4 +228,17 @@ def filterrific_params
       with_status: [],
       with_owner: [])
   end
+
+  def search_protocol_merges(protocol_id)
```

Comment:
Can you give a short explanation of what you added in this controller? I'm struggling to follow what it's doing. Thanks!